src/services/expenses.service.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+let expenses = [];
+
+const getAll = () => expenses;
+
+const add = expense => {
+  expenses.push(expense);
+};
+
+const getById = id => (
+  expenses.find(expense => +expense.id === +id)
```

Comment:
```suggestion
  expenses.find(expense => Number(expense.id) === Number(id))
```

The same in other places in code